conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -517,3 +518,26 @@ def generate(self):
     client.save({"conanfile.py": conanfile})
     client.run("install .", assert_error=True)
     assert "You can only pass native=True if you're cross-building" in client.out
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="requires Win")
```

Comment:
I might be missing something but why does this only work in Windows? Couldnt the profile just specify the conf?